python/qpid_dispatch_internal/tools/command.py

```diff
@@ -261,9 +261,8 @@ def opts_url(opts):
     url = Url(opts.bus)
 
     # If the options indicate SSL, make sure we use the amqps scheme.
-    if opts.ssl_certificate or opts.ssl_trustfile:
+    if opts.ssl_certificate or opts.ssl_trustfile or opts.bus.startswith("amqps"):
```

Comment:
This looks wrong shouldn't it be ...startswith("amqps:")? Otherwise what happens if you connect to amqps.com?